sed -e 's%set(CMAKE_MODULES_INSTALL_DIR .*%set(CMAKE_MODULES_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")%g' -i */CMakeLists.txt
diff '--color=auto' -ur linphone-sdk-5.5.15.orig/bcg729/CMakeLists.txt linphone-sdk-5.5.15/bcg729/CMakeLists.txt
--- linphone-sdk-5.5.15.orig/bcg729/CMakeLists.txt	2024-05-16 09:14:24.000000000 +0200
+++ linphone-sdk-5.5.15/bcg729/CMakeLists.txt	2026-08-10 11:44:40.999538383 +0200
@@ -77,7 +77,7 @@
 
 
 include(CMakePackageConfigHelpers)
-set(CMAKE_MODULES_INSTALL_DIR "${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}/cmake")
+set(CMAKE_MODULES_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")
 configure_package_config_file("${PROJECT_NAME}Config.cmake.in" "${PROJECT_BINARY_DIR}/${PROJECT_NAME}Config.cmake"
 	INSTALL_DESTINATION "${CMAKE_MODULES_INSTALL_DIR}"
 	NO_SET_AND_CHECK_MACRO
diff '--color=auto' -ur linphone-sdk-5.5.15.orig/bcmatroska2/CMakeLists.txt linphone-sdk-5.5.15/bcmatroska2/CMakeLists.txt
--- linphone-sdk-5.5.15.orig/bcmatroska2/CMakeLists.txt	2025-04-14 11:07:41.000000000 +0200
+++ linphone-sdk-5.5.15/bcmatroska2/CMakeLists.txt	2026-08-10 11:44:40.999538383 +0200
@@ -74,7 +74,7 @@
 add_subdirectory(build)
 
 include(CMakePackageConfigHelpers)
-set(CMAKE_MODULES_INSTALL_DIR "${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}/cmake")
+set(CMAKE_MODULES_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")
 configure_package_config_file("${PROJECT_NAME}Config.cmake.in" "${PROJECT_BINARY_DIR}/${PROJECT_NAME}Config.cmake"
 	INSTALL_DESTINATION "${CMAKE_MODULES_INSTALL_DIR}"
 	NO_SET_AND_CHECK_MACRO
Only in linphone-sdk-5.5.15.orig/bcmatroska2: bcmatroska2.spec
diff '--color=auto' -ur linphone-sdk-5.5.15.orig/bctoolbox/CMakeLists.txt linphone-sdk-5.5.15/bctoolbox/CMakeLists.txt
--- linphone-sdk-5.5.15.orig/bctoolbox/CMakeLists.txt	2026-08-06 09:00:06.000000000 +0200
+++ linphone-sdk-5.5.15/bctoolbox/CMakeLists.txt	2026-08-10 11:44:41.000538363 +0200
@@ -155,7 +155,7 @@
 endif()
 
 include(CMakePackageConfigHelpers)
-set(CMAKE_MODULES_INSTALL_DIR "${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}/cmake")
+set(CMAKE_MODULES_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")
 configure_package_config_file("cmake/${PROJECT_NAME}Config.cmake.in" "${PROJECT_BINARY_DIR}/${PROJECT_NAME}Config.cmake"
 	INSTALL_DESTINATION "${CMAKE_MODULES_INSTALL_DIR}"
 	PATH_VARS CMAKE_MODULES_INSTALL_DIR
diff '--color=auto' -ur linphone-sdk-5.5.15.orig/bcunit/CMakeLists.txt linphone-sdk-5.5.15/bcunit/CMakeLists.txt
--- linphone-sdk-5.5.15.orig/bcunit/CMakeLists.txt	2026-08-06 09:00:06.000000000 +0200
+++ linphone-sdk-5.5.15/bcunit/CMakeLists.txt	2026-08-10 11:44:41.000538363 +0200
@@ -91,7 +91,7 @@
 
 
 include(CMakePackageConfigHelpers)
-set(CMAKE_MODULES_INSTALL_DIR "${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}/cmake")
+set(CMAKE_MODULES_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")
 configure_package_config_file("${PROJECT_NAME}Config.cmake.in" "${PROJECT_BINARY_DIR}/${PROJECT_NAME}Config.cmake"
 	INSTALL_DESTINATION "${CMAKE_MODULES_INSTALL_DIR}"
 	NO_SET_AND_CHECK_MACRO
diff '--color=auto' -ur linphone-sdk-5.5.15.orig/belcard/CMakeLists.txt linphone-sdk-5.5.15/belcard/CMakeLists.txt
--- linphone-sdk-5.5.15.orig/belcard/CMakeLists.txt	2026-08-06 09:00:06.000000000 +0200
+++ linphone-sdk-5.5.15/belcard/CMakeLists.txt	2026-08-10 11:44:41.000538363 +0200
@@ -105,7 +105,7 @@
 endif()
 
 include(CMakePackageConfigHelpers)
-set(CMAKE_MODULES_INSTALL_DIR "${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}/cmake")
+set(CMAKE_MODULES_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")
 configure_package_config_file("${PROJECT_NAME}Config.cmake.in" "${PROJECT_BINARY_DIR}/${PROJECT_NAME}Config.cmake"
 	INSTALL_DESTINATION "${CMAKE_MODULES_INSTALL_DIR}"
 	NO_SET_AND_CHECK_MACRO
diff '--color=auto' -ur linphone-sdk-5.5.15.orig/belr/CMakeLists.txt linphone-sdk-5.5.15/belr/CMakeLists.txt
--- linphone-sdk-5.5.15.orig/belr/CMakeLists.txt	2026-08-06 09:00:06.000000000 +0200
+++ linphone-sdk-5.5.15/belr/CMakeLists.txt	2026-08-10 11:44:41.001538343 +0200
@@ -81,7 +81,7 @@
 endif()
 
 include(CMakePackageConfigHelpers)
-set(CMAKE_MODULES_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")
+set(CMAKE_MODULES_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")
 configure_package_config_file("${PROJECT_NAME}Config.cmake.in" "${PROJECT_BINARY_DIR}/${PROJECT_NAME}Config.cmake"
 	INSTALL_DESTINATION "${CMAKE_MODULES_INSTALL_DIR}"
 	NO_SET_AND_CHECK_MACRO
diff '--color=auto' -ur linphone-sdk-5.5.15.orig/bzrtp/CMakeLists.txt linphone-sdk-5.5.15/bzrtp/CMakeLists.txt
--- linphone-sdk-5.5.15.orig/bzrtp/CMakeLists.txt	2026-08-06 09:00:06.000000000 +0200
+++ linphone-sdk-5.5.15/bzrtp/CMakeLists.txt	2026-08-10 11:44:41.001538343 +0200
@@ -117,7 +117,7 @@
 endif()
 
 include(CMakePackageConfigHelpers)
-set(CMAKE_MODULES_INSTALL_DIR "${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}/cmake")
+set(CMAKE_MODULES_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")
 configure_package_config_file("cmake/${PROJECT_NAME}Config.cmake.in" "${PROJECT_BINARY_DIR}/${PROJECT_NAME}Config.cmake"
 	INSTALL_DESTINATION "${CMAKE_MODULES_INSTALL_DIR}"
 	NO_SET_AND_CHECK_MACRO
diff '--color=auto' -ur linphone-sdk-5.5.15.orig/liblinphone/CMakeLists.txt linphone-sdk-5.5.15/liblinphone/CMakeLists.txt
--- linphone-sdk-5.5.15.orig/liblinphone/CMakeLists.txt	2026-08-06 09:00:06.000000000 +0200
+++ linphone-sdk-5.5.15/liblinphone/CMakeLists.txt	2026-08-10 11:44:41.002538323 +0200
@@ -471,7 +471,7 @@
 endif()
 
 include(CMakePackageConfigHelpers)
-set(CMAKE_MODULES_INSTALL_DIR "${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}/cmake")
+set(CMAKE_MODULES_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")
 configure_package_config_file("cmake/${PROJECT_NAME}Config.cmake.in" "${PROJECT_BINARY_DIR}/${PROJECT_NAME}Config.cmake"
 	INSTALL_DESTINATION "${CMAKE_MODULES_INSTALL_DIR}"
 	PATH_VARS LIBLINPHONE_PLUGINS_DIR
diff '--color=auto' -ur linphone-sdk-5.5.15.orig/lime/CMakeLists.txt linphone-sdk-5.5.15/lime/CMakeLists.txt
--- linphone-sdk-5.5.15.orig/lime/CMakeLists.txt	2026-08-06 09:00:06.000000000 +0200
+++ linphone-sdk-5.5.15/lime/CMakeLists.txt	2026-08-10 11:44:41.002538323 +0200
@@ -135,7 +135,7 @@
 endif()
 
 include(CMakePackageConfigHelpers)
-set(CMAKE_MODULES_INSTALL_DIR "${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}/cmake")
+set(CMAKE_MODULES_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")
 configure_package_config_file("cmake/${PROJECT_NAME}Config.cmake.in" "${PROJECT_BINARY_DIR}/${PROJECT_NAME}Config.cmake"
 	INSTALL_DESTINATION "${CMAKE_MODULES_INSTALL_DIR}"
 	NO_SET_AND_CHECK_MACRO
diff '--color=auto' -ur linphone-sdk-5.5.15.orig/mediastreamer2/CMakeLists.txt linphone-sdk-5.5.15/mediastreamer2/CMakeLists.txt
--- linphone-sdk-5.5.15.orig/mediastreamer2/CMakeLists.txt	2026-08-06 09:00:06.000000000 +0200
+++ linphone-sdk-5.5.15/mediastreamer2/CMakeLists.txt	2026-08-10 11:44:41.003538303 +0200
@@ -775,7 +775,7 @@
 endif()
 
 include(CMakePackageConfigHelpers)
-set(CMAKE_MODULES_INSTALL_DIR "${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}/cmake")
+set(CMAKE_MODULES_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")
 configure_package_config_file("cmake/${PROJECT_NAME}Config.cmake.in" "${PROJECT_BINARY_DIR}/${PROJECT_NAME}Config.cmake"
 	INSTALL_DESTINATION "${CMAKE_MODULES_INSTALL_DIR}"
 	PATH_VARS MS2_PLUGINS_DIR
diff '--color=auto' -ur linphone-sdk-5.5.15.orig/ortp/CMakeLists.txt linphone-sdk-5.5.15/ortp/CMakeLists.txt
--- linphone-sdk-5.5.15.orig/ortp/CMakeLists.txt	2026-08-06 09:00:06.000000000 +0200
+++ linphone-sdk-5.5.15/ortp/CMakeLists.txt	2026-08-10 11:44:41.005538263 +0200
@@ -197,7 +197,7 @@
 endif()
 
 include(CMakePackageConfigHelpers)
-set(CMAKE_MODULES_INSTALL_DIR "${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}/cmake")
+set(CMAKE_MODULES_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")
 configure_package_config_file("cmake/${PROJECT_NAME}Config.cmake.in" "${PROJECT_BINARY_DIR}/${PROJECT_NAME}Config.cmake"
 	INSTALL_DESTINATION "${CMAKE_MODULES_INSTALL_DIR}"
 	NO_SET_AND_CHECK_MACRO
diff '--color=auto' -ur linphone-sdk-5.5.15.orig/postquantumcryptoengine/CMakeLists.txt linphone-sdk-5.5.15/postquantumcryptoengine/CMakeLists.txt
--- linphone-sdk-5.5.15.orig/postquantumcryptoengine/CMakeLists.txt	2026-08-06 09:00:06.000000000 +0200
+++ linphone-sdk-5.5.15/postquantumcryptoengine/CMakeLists.txt	2026-08-10 11:44:41.006538243 +0200
@@ -74,7 +74,7 @@
 endif()
 
 include(CMakePackageConfigHelpers)
-set(CMAKE_MODULES_INSTALL_DIR "${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}/cmake")
+set(CMAKE_MODULES_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")
 configure_package_config_file("${PROJECT_NAME}Config.cmake.in" "${PROJECT_BINARY_DIR}/${PROJECT_NAME}Config.cmake"
 	INSTALL_DESTINATION "${CMAKE_MODULES_INSTALL_DIR}"
 	NO_SET_AND_CHECK_MACRO
